src/iTunesLibrary/ITCompat.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace iTunesLibrary
+{
+#if !XAMCORE_2_0
+    public partial class ITLibrary {
+
+        [Obsolete ("Does not return a valid instance on macOS 14.")]

```

Comment:
There's no need to mention the version - it's not clear if this a change in macOS 10.15 (or if it was already the case) and the type is new in macOS 10.14.

Also it's not `macOS 14` ;-) even if I wish that least year `X` removal (from Mac OSX) also removed the `10`

-> `[Obsolete ("This constructor does not create a valid instance of the type.")]`

^ it's an existing string (so it won't increase the metadata size)